src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationUseStyleCheck.java

```diff
@@ -126,48 +126,98 @@
  * Example:
  * </p>
  * <pre>
+ * &#64;SuppressWarnings("unchecked") // OK
  * &#64;Deprecated // OK
- * &#64;SomeArrays(pooches={DOGS.LEO}) // Violation - COMPACT_NO_ARRAY
- * &#64;SuppressWarnings({""}) // Violation - COMPACT_NO_ARRAY
+ * &#64;SomeArrays({"unchecked","unused"}) // OK
  * public class TestOne
  * {
  *
  * }
  *
- * &#64;SomeArrays(pooches={DOGS.LEO}, um={}, test={"bleh"}) // Violation - COMPACT_NO_ARRAY
- * &#64;SuppressWarnings("") // OK
- * &#64;Deprecated() // Violation - cannot have closing parenthesis
+ * &#64;SuppressWarnings(value={"unchecked"}) // Violation - COMPACT_NO_ARRAY
```

Comment:
Instead of COMPACT_NO_ARRAY, let's use wording "no parameter should be referenced"

Reply:
Hi, @romani 
Thank you for reviewing my code. And I will reply to two points you mentioned to me.

1: Please update existing configs to have more properties to show them with more values of each property.
I got it. I am planning to cover all possible cases. Is it too many?

Plan
Add "compact" and "ignore" configs to elementStyle
Add "ignore" to closingParens
Add "ignore" to trailingArrayComma
 
So, there will be four additional configs with examples.

2: Instead of COMPACT_NO_ARRAY, let's use wording "no parameter should be referenced"
I got it. I will change the wording.

After I finish all changes you mentioned, I will update the PR description.

And I have an additional question about how I should communicate with people. Usually, when I am working on the issue, I propose my plan before I will start working on it (Just like what I did above) to confirm whether I am on the right track or not. To the reviewer, is it better to send PR without any plan beforehand since we can talk a more exact context? Or is it better to propose a plan first and then start working on it?

Reply:
>. Is it too many?

Limit it to 4 examples with not default configs.

Each reply from maintainer can take 1-2 days, usually better to send simple change without plan, and do planning sync only on complicated update.

Reply:
Your update is simple, just send new code.

Reply:
@romani 
Thank you for your suggestion! I will limit them to 4 examples by adding multiple properties to the example.